### PR TITLE
[landscape] Add check for LP#2081056.

### PR DESCRIPTION
**Pull request by pponnuvel**
Closes: #977.

### CODE REVIEW

**Review by nicolasbock — APPROVED** [review]

File: hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+checks:
+  has_2081056:
+    input:
+      path: 'var/log/landscape-server/job-handler.log*'
```
Comment (dosaboy) [inline hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml:4]: don't need the * as hotsos will take care of that for you

File: hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    expr: ".+The lock file './db/lockfile' already exists. There might be another instance with .+"
```
Comment (dosaboy) [inline hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml:5]: dont need to end the expr with .+

Comment (pponnuvel) [reply hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml]: Done

File: hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml (hunk 1 above)
Comment (pponnuvel) [reply hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml]: Done

**Review by dosaboy — APPROVED** [review]

**Pull request merged**